benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaConsumerBenchmarks.scala

```diff
@@ -69,7 +69,7 @@ object ReactiveKafkaConsumerBenchmarks extends LazyLogging {
         batch.updated(elem)
 
       }
-      .mapAsync(1) { m =>
+      .mapAsync(3) { m =>
```

Comment:
If you process message 1 and 2 and commit them in order of 2 and 1 then on restart you will start processing from last committed offset (1) and process message 2 once again